tools/harness-simulation/harness/Web/data/deviceInputFields.xml

```diff
@@ -0,0 +1,11 @@
+<DEVICE_FIELDS>
+    <DEVICE name="OpenThread_Sim" thumbnail="OpenThread.png" description = "OpenThread Simulation" THCI="OpenThread_Sim">
+        <Hardware thumbnail="Raspberry_Pi_4B.jpg", description="OpenThread Simulation FTD"></Hardware>
```

Comment:
```suggestion
        <Hardware thumbnail="Raspberry_Pi_4B.jpg" description="OpenThread Simulation FTD"></Hardware>
```

Reply:
Thanks for pointing out the mistake. And I have added the image for OpenThread Simulation running on POSIX.